thundra/integrations/botocore.py

```diff
@@ -196,44 +175,41 @@ def get_operation_name(self, wrapped, instance, args, kwargs):
             if arn != 'N/A':
                 self.topicName = arn.split(':')[-1]
             else:
-                self.topicName = Constants.AWS_SERVICE_REQUEST
+                self.topicName = constants.AWS_SERVICE_REQUEST
```

Comment:
Return empty string instead of `AWS_SERVICE_REQUEST` constant if the topic name is not available

Reply:
But be sure that in case of empty topic name, operation name is not empty but `AWS_SERVICE_REQUEST` constant